entrypoint.sh

```diff
@@ -5,7 +5,7 @@ case "$1" in
     develop)
         echo "Running Development Server"
         sleep 20
-        exec yarn run dev | bunyan
+        exec yarn start | bunyan
```

Comment:
Not really sure what this does and why it's needed, up to you to decide if it stays or goes

Reply:
This makes it work.  The package.json doesn't have `dev` option.